worker/src/main/scala/org/apache/celeborn/service/deploy/worker/WorkerSource.scala

```diff
@@ -64,6 +65,13 @@ class WorkerSource(conf: CelebornConf) extends AbstractSource(conf, MetricsSyste
     val metricNameWithLabel = metricNameWithCustomizedLabels(metricsName, Map.empty)
     namedCounters.get(metricNameWithLabel).counter.getCount
   }
+
+  def incAppConnectionCounter(shuffleKey: String): Unit = {
+    val applicationLabel = Map("applicationId" -> Utils.splitShuffleKey(shuffleKey)._1)
+    addCounter(WorkerSource.ACTIVE_CONNECTION_COUNT, applicationLabel)
```

Comment:
ACTIVE_CONNECTION means that a new networker connection is established not a logic connection. Celeborn client and worker will reuse connections, so I don't think this PR should use ACTIVE_CONNECTION_COUNT here.